whole-tale/ngx-dashboard
# fix: disable "Restart Tale" when Tale is stopped

## Problem
"Restart Tale" is offered even when Tale is not running

Fixes #199 

## Approach
* Hide this option from the dropdown
* Also make the icon spacing consistent on this dropdown (it was bothering me)

NOTE: I tried simply disabling this option, but that doesn't allow me to show a tooltip since it is `disabled` ... so there was no way how to let the user know why the item was disabled.

## How to Test
Prerequisites: at least one Tale created, not running

1. Checkout this branch locally, rebuild the dashboard
2. Login to the WholeTale Dashboard
3. Navigate to Run Tale for a stopped Tale
4. Expand the dropdown at the top-right
    * You should **not** see the "Restart Tale" option listed here
5. Run the Tale, wait for Tale to fully start (e.g. `instance.status === 1`)
6. Expand the dropdown again
    * You should see the "Restart Tale" option is now listed here
